src/Raven.Client/Http/DatabaseTopologyLocalCache.cs

```diff
@@ -109,10 +112,18 @@ private static void WriteNode(BlittableJsonTextWriter writer, ServerNode node, J
             writer.WriteString(context.GetLazyString(node.Url));
 
             writer.WriteComma();
-
             writer.WritePropertyName(context.GetLazyString(nameof(ServerNode.Database)));
             writer.WriteString(context.GetLazyString(node.Database));
 
+            // ClusterTag and ServerRole included for debugging purpose only
+            writer.WriteComma();
+            writer.WritePropertyName(context.GetLazyString(nameof(ServerNode.ClusterTag)));
+            writer.WriteString(context.GetLazyString(node.ClusterTag));
+
+            writer.WriteComma();
+            writer.WritePropertyName(context.GetLazyString(nameof(ServerNode.ServerRole)));
+            writer.WriteString(context.GetLazyString(node.ServerRole.ToString()));
+
```

Comment:
Also include time and etag of the topology.

Reply:
Done

Reply:
Do we have an etag also for the cluster topology?